src/main/java/co/cask/http/NettyHttpService.java

```diff
@@ -73,6 +73,7 @@
   private final ChannelGroup channelGroup;
   private final HttpResourceHandler resourceHandler;
   private final Function<ChannelPipeline, ChannelPipeline> pipelineModifier;
+  private final ExceptionHandler exceptionHandler;
```

Comment:
This can be a local field in the constructor. You should see a warning in IntelliJ.